Updates to language statistics as per Trello task "Fix the language tag analysis."

J-Jinn opened the pull request:
I hope I understood what you meant by the Trello task.  If not can continue revising and editing as necessary.

"slo_twitter_data_analysis.py" and "slo-twitter-data-analysis-language-statistics.ipynb" are the two files that contains modified/added code.

Commit:
Updates to language statistics as per Trello task "Fix the language tag analysis."

kvlinden commented:
We need the percentage of tweets marked as English by either Twitter OR SpaCy. That percentage should be at least as high as than either 96.43% and 95.85%. That's the way the code works (or used to work): if TwitterLang="en" then accept, else if SpacyLang="en", then accept. Only reject if both Twitter and Spacy mark the tweet as non-Engish.

Please add this stat (Twitter OR Spacy) to the language-statistics notebook and add a summary ("we accept as English anything marked as such by either Twitter or Spacy) to the TOC notebook. And I'd like you to double-check the code to make sure that this is the way we're doing it.

Commit:
Updated language statistics Jupyter Notebook as per e-mail explanation.

Also updated "slo_twitter_data_analysis.py" with relevant code for language statistics.
Also updated "dataset_processor_adapted.py" to fix Tweet associated company count.

"company_derived" now states "none" for no associated companies for a Tweet.

kvlinden commented:
That's better, but the TOC summary should say that we accept 97.79% as English ("en" by either Twitter or SpaCy) and 2.21% as non-English ("en" by neither Twitter nor SpaCy).

Commit:
Rewrote summary in table of contents Jupyter Notebook for language statistics as per reviewer commentary.
